server/src/main/java/org/candlepin/resource/OwnerProductResource.java

```diff
@@ -202,7 +207,10 @@ public ProductCertificate getProductCertificate(@PathParam("owner_key") String o
      */
     @POST
     @Produces(MediaType.APPLICATION_JSON)
-    public Product createProduct(@PathParam("owner_key") String ownerKey, Product product) {
+    public Product createProduct(
+        @Verify(Owner.class) @PathParam("owner_key") String ownerKey,
+        Product product) {
```

Comment:
This one worries me a bit as well. An org admin might have write perms for an Owner, should they be able to create products? I think product creation might have been super admin only, forcing normal users to come in via a manifest import, or Katello UI which operates as a super admin when calling candlepin.